Cargo.toml

```diff
@@ -18,45 +18,45 @@ all-features = true
 # These two are generally really common simple dependencies so it does not seem
 # much of a point to optimize these, but these could in theory be removed for
 # an indirection through std::Formatter.
-dtoa = "0.4"
-itoa = "0.4.3"
+dtoa = "0.4.6"
+itoa = "0.4.6"
 
 # This is a dependency that already exists in url
-percent-encoding = "2.1"
+percent-encoding = "2.1.0"
 
 # We need this for redis url parsing
-url = "2.1"
+url = "2.1.1"
 
 # We need this for script support
-sha1 = { version = ">= 0.2, < 0.7", optional = true }
+sha1 = { version = "0.6.0", optional = true }
```

Comment:
This just forces users to the 0.6 version, which could lead to multiple sha1 crates in the dependency graph. Not a big problem but I see no reason to change this to be more restrictive.